workshops/templates/workshops/instructors.html

```diff
@@ -6,18 +6,13 @@
     {% breadcrumb_active title %}
 {% endblock %}
 
+{% load crispy_forms_tags %}
 {% block content %}
-<form action="{% url 'instructors' %}" method="post">
-    {% csrf_token %}
-    <table class="table table-striped">
-    {{ form.as_table }}
-    </table>
-    <input type="submit" value="Submit" />
-</form>
+    {% crispy form form_helper %}
 {% if persons == None %}
 {% elif persons %}
     <table class="table table-striped">
-        <tr>
+    <tr>
```

Comment:
Drive-by whitespace change.  You want the `<tr>` indented beyond it's wrapping `<table>`, no?

And personally, I'd use `<thead>`, `<tbody>`, etc., but that's beyond the scope of this PR.